pkgs/applications/video/mpv/default.nix

```diff
@@ -39,7 +39,7 @@
 , libpngSupport      ? true,           libpng        ? null
 , pulseSupport       ? config.pulseaudio or stdenv.isLinux, libpulseaudio ? null
 , rubberbandSupport  ? stdenv.isLinux, rubberband    ? null
-, sambaSupport       ? stdenv.isLinux, samba         ? null
+, sambaSupport       ? false,          samba         ? null
```

Comment:
```suggestion
# NOTE: samba support should be removed on the next mpv release, see also:
# https://github.com/NixOS/nixpkgs/pull/89145#issuecomment-636424362
# Please remove this line on the next mpv release.
, sambaSupport       ? false,          samba         ? null
```

Reply:
Applied and amended. Thanks for the suggestion!